### ISSUE TITLE
Accessibility of rule details button.

### ISSUE THREAD

**Issue by joedolson** [equalizedigital/accessibility-checker#54]
The `button` for expnading rule details (.edac-details-rule-title-arrow) has no accessible name:

![image](https://user-images.githubusercontent.com/1686451/200086208-bfa2ed0d-1a88-4f5e-a05b-6d12edf6b570.png)

It also has a very poor focus state: 

![image](https://user-images.githubusercontent.com/1686451/200086327-3efad27e-9fc2-4aa0-adbb-114aa772bb8f.png)

(Top button is focused; bottom two are not.)

**Comment by SteveJonesDev** [comment]
Updated since version 1.3.10
